docs/en/sql-reference/functions/random-functions.md

```diff
@@ -23,6 +23,68 @@ Uses a linear congruential generator.
 
 ## randConstant {#randconstant}
 
-Returns a pseudo-random UInt32 number, The value is one for different blocks.
+Produces a constant column with a random value.
+
+The function is intended for service purposes.
```

Comment:
Let's delete it. I asked developers, they don't know why it is the "service" function.